test/support/database.js

```diff
@@ -1,3 +1,5 @@
+const ANALYTICS_DATA_TABLE_NAME = "analytics_data"
```

Comment:
Not yet b/c we're exporting functions. I have a PR to export objects and I have few changes like this I want to come behind that PR with.